lib/api/controllers/documentController.js

```diff
@@ -49,7 +49,7 @@ function DocumentController (kuzzle) {
    * @returns {Promise<Object>}
    */
   this.scroll = function documentScroll (request) {
-    if (!request.input.args.scroll) {
+    if (!request.input.body || !request.input.body.scroll) {
       throw new BadRequestError('document:scroll must specify an argument "scroll"');
     }
     if (!request.input.args.scrollId) {
```

Comment:
If I follow the documentation, we should also replace `request.input.args.scrollId` by `request.input.body.scrollId` ?